vyxal/elements.py

```diff
@@ -443,6 +444,7 @@ def process_element(
     "k□": process_element("[[0,1],[1,0],[0,-1],[-1,0]]", 0),
     "kṘ": process_element('"IVXLCDM"', 0),
     "k•": process_element('["qwertyuiop","asdfghjkl","zxcvbnm"]', 0),
+    "¨w": ("time.sleep((_:=pop(stack, 1, ctx)) if type(_)==type(1) else 0)", 1),
```

Comment:
This shouldn't push 0. How about `lhs = pop(stack, 1, ctx)\nif vy_type(lhs) == NUMBER_TYPE: time.sleep(int(lhs))`

Reply:
It waits 0